src/components/Select/AutosuggestMulti/SuggestionTag/__tests__/SuggestionTag.js

```diff
@@ -10,11 +10,7 @@ describe('SuggestionTag', () => {
         expect(wrapper.find('button')).toHaveLength(0);
     });
     it('should render correctly with styles modifier', () => {
-        const wrapper = mount(
-            <SuggestionTag isBounded isStretched>
-                tag
-            </SuggestionTag>
-        );
+        const wrapper = mount(<SuggestionTag width="block">tag</SuggestionTag>);
```

Comment:
these tests are not executed. Need to change the file name to include `spec`

Reply:
Done

Reply:
it is not in the commit. The whole file got deleted. :-(